### PR TITLE
Link updates

### PR DESCRIPTION
**Pull request by estrattonbailey**

### CODE REVIEW

File: src/alf/atoms.ts
Comment (estrattonbailey) [inline src/alf/atoms.ts:1]: Just adding atoms to this PR since some new ones were used

File: src/components/Link.tsx [hunk 1]
```diff
@@ -46,13 +48,21 @@ type BaseLinkProps = Pick<
    * Note: atm this only works for `InlineLink`s with a string child.
    */
   warnOnMismatchingTextChild?: boolean
+
+  /**
+   * Callback for when the link is pressed.
+   *
+   * DO NOT use this for navigation, that's what the `to` prop is for.
+   */
+  onPress?: (e: GestureResponderEvent) => void
```
Comment (estrattonbailey) [inline src/components/Link.tsx:57]: Helpful for closing modals 🙃

File: src/components/Link.tsx [hunk 2]
```diff
@@ -245,27 +263,27 @@ export function InlineLink({
...
-          dataSet: stringChildren
-            ? {}
-            : {
-                // default to no underline, apply this ourselves
-                noUnderline: '1',
-              },
+          dataSet: {
+            // default to no underline, apply this ourselves
+            noUnderline: '1',
+          },
```
Comment (estrattonbailey) [inline src/components/Link.tsx:286]: Children aren't a string if we use `Trans`, so this assumption was breaking.

File: src/view/screens/Storybook/Links.tsx [hunk 3]
```diff
@@ -19,9 +19,14 @@ export function Links() {
...
         </InlineLink>
-        <InlineLink to="https://bsky.social" style={[a.text_md]}>
+        <InlineLink to="https://bsky.social" style={[a.text_md, t.atoms.text]}>
           <H3>External with custom children</H3>
         </InlineLink>
+        <InlineLink
+          to="https://bsky.social"
+          style={[a.text_md, t.atoms.text_contrast_low]}>
+          External with custom children
+        </InlineLink>
```
Comment (estrattonbailey) [inline src/view/screens/Storybook/Links.tsx:29]: Just examples

**Review by pfrazee — APPROVED** [review]
LGTM. You dropped the `download` attr I added but it wasnt working anyway

**Pull request merged**